### DISCUSSION TITLE
Effortless Android Library Binding

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#80] [Announcements]
# Effortless Android Library Binding

Bind Java libraries directly from Maven repositories

https://vladislavantonyuk.github.io/articles/Effortless-Android-Library-Binding/

**Comment by matt-goldman** [comment]
Great write up. I've dabbled with binding Android libraries before and found it painful. This looks like a much simpler approach!

**Reply by VladislavAntonyuk** [reply]
To be honest this is my first successful binding. It would be great if all libraries had such good description of compilation errors.
